docs/microservices.json

```diff
@@ -706,15 +671,15 @@
             "version": null
         },
         {
-            "id": "evidence-client-api",
-            "name": "Client API",
+            "id": "div-evidence-management-client-api",
+            "name": "Divorce Evidence Management Client API",
```

Comment:
can this have a new line please? it'll be really long in the graph

Reply:
![screenshot from 2018-06-20 09-28-06](https://user-images.githubusercontent.com/3073565/41646660-546024ac-746c-11e8-8a3e-0ac59aa1ca0b.png)

Reply:
yea, looks more concise with borders. graph is not messy and you clearly see actual interactions among services